spec/factories/supplier.rb

```diff
@@ -20,4 +21,9 @@
     factory :shared_supplier, class: SharedSupplier
   end
 
+  factory :supplier_category do
+    sequence(:name) { |n| Faker::Lorem.characters(number: rand(2..12)) + " ##{n}" }
+    financial_transaction_class
+  end
+
```

Comment:
other PR

Reply:
nope, `SupplierCategory` is mandatory in the model (please check out the multiple commits of this PR)